packages/flutter_tools/pubspec.yaml

```diff
@@ -60,15 +60,14 @@ dependencies:
 
   standard_message_codec: 0.0.1+3
 
-  _fe_analyzer_shared: 62.0.0 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
-  analyzer: 6.0.0 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
+  _fe_analyzer_shared: 61.0.0 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
+  analyzer: 5.13.0 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
```

Comment:
@jason-simmons what happened here?

Reply:
The latest fe_analyzer_shared depends on a package dart_internal which has not been updated for 2.12 compatibility. So either we roll back the fe_analyzer_shared dep or disable dart sdk rolls.

Since this version has zero impact on what the dart analyzer or CLI compiler use, it doesn't really matter that we're rolling back a version.

Reply:
@scheglov do you have any idea why this might have reverted?

Reply:
(it seems like a mistake that the dart_internal package was added in the first place but 🤷 )

Reply:
ahh, I see